lib/src/dom/pretty_dom.dart

```diff
@@ -98,4 +101,13 @@ class PrettyDomOptions {
   /// {@endtemplate}
   external bool? get min;
   external set min(bool? value);
+
+  /// {@template prettyDomOptionsFilterNode}
+  /// ### [filterNode]
+  /// By default, `<style />`, `<script />` and comment nodes are ignored.
+  /// You can configure this behavior by passing a custom `filterNode` function that should return true for every
+  /// node that you wish to include in the output.
+  /// {@endtemplate}
+  external bool? Function(Node?)? get filterNode;
+  external set filterNode(bool? Function(Node?)? value);
```

Comment:
Is there a reason the arg and return value aren't typed as non-nullable like in [the TS type](https://github.com/testing-library/dom-testing-library/blob/v8.20.1/types/pretty-dom.d.ts#L8)?
```suggestion
  external bool Function(Node)? get filterNode;
  external set filterNode(bool Function(Node)? value);
```